builtin/logical/pki/path_resign_crls.go

```diff
@@ -0,0 +1,271 @@
+			},
+			crlNumberParam: {
+				Type:        framework.TypeInt,
+				Description: `The sequence number to be written within the CRL Number extension.`,
+			},
+			deltaCrlBaseNumberParam: {
+				Type: framework.TypeInt,
+				Description: `Using a zero or greater value specifies the base CRL revision number to encode within
+ a Delta CRL indicator extension, otherwise the extension will not be added.`,
+				Default: -1,
+			},
+			nextUpdateParam: {
+				Type: framework.TypeString,
+				Description: `The amount of time the generated CRL should be
+valid; defaults to 72 hours.`,
+				Default: defaultCrlConfig.Expiry,
+			},
+			crlsParam: {
+				Type:        framework.TypeStringSlice,
+				Description: `A list of PEM encoded CRLs to combine, originally signed by the requested issuer.`,
+			},
+			formatParam: {
+				Type: framework.TypeString,
+				Description: `The format of the combined CRL, can be "pem" or "der". If "der", the value will be
+base64 encoded. Defaults to "pem".`,
+				Default: "pem",
+			},
+		},
+		Operations: map[logical.Operation]framework.OperationHandler{
+			logical.UpdateOperation: &framework.PathOperation{
+				Callback: b.pathUpdateResignCrlsHandler,
+			},
+		},
+
+		HelpSynopsis: `Combine and sign with the provided issuer different CRLs`,
+		HelpDescription: `Provide two or more PEM encoded CRLs signed by the issuer,
+ normally from separate Vault clusters to be combined and signed.`,
+	}
+}
+
+func (b *backend) pathUpdateResignCrlsHandler(ctx context.Context, request *logical.Request, data *framework.FieldData) (*logical.Response, error) {
+	if b.useLegacyBundleCaStorage() {
+		return logical.ErrorResponse("This API cannot be used until the migration has completed"), nil
+	}
+
+	issuerRef := getIssuerRef(data)
+	crlNumber := data.Get(crlNumberParam).(int)
+	deltaCrlBaseNumber := data.Get(deltaCrlBaseNumberParam).(int)
+	nextUpdateStr := data.Get(nextUpdateParam).(string)
+	rawCrls := data.Get(crlsParam).([]string)
+
+	format, err := getCrlFormat(data.Get(formatParam).(string))
+	if err != nil {
+		return logical.ErrorResponse(err.Error()), nil
+	}
+
+	nextUpdateOffset, err := time.ParseDuration(nextUpdateStr)
+	if err != nil {
+		return logical.ErrorResponse("invalid value for %s: %v", nextUpdateParam, err), nil
+	}
+
+	if nextUpdateOffset <= 0 {
+		return logical.ErrorResponse("%s parameter must be greater than 0", nextUpdateParam), nil
+	}
+
+	if crlNumber < 0 {
+		return logical.ErrorResponse("%s parameter must be 0 or greater", crlNumberParam), nil
+	}
+	if deltaCrlBaseNumber < -1 {
+		return logical.ErrorResponse("%s parameter must be -1 or greater", deltaCrlBaseNumberParam), nil
+	}
+
+	if issuerRef == "" {
+		return logical.ErrorResponse("%s parameter cannot be blank", issuerRefParam), nil
+	}
+
+	providedCrls, err := decodePemCrls(rawCrls)
+	if err != nil {
+		return logical.ErrorResponse(err.Error()), nil
+	}
+
+	sc := b.makeStorageContext(ctx, request.Storage)
+	caBundle, err := getCaBundle(sc, issuerRef)
```

Comment:
I forgot to catch this on the earlier PR, but do you want to catch issuer usage here and ensure we have CRL signing usage too?

Reply:
Well it is validated within `getCaBundle` that we have `CRLSigningUsage`?

Reply:
Oh, I'm stupid and can't read code. You're right.